src/ParallelAlgorithms/Interpolation/Events/InterpolateWithoutInterpComponent.hpp

```diff
@@ -121,32 +122,39 @@ class InterpolateWithoutInterpComponent<VolumeDim, InterpolationTargetTag,
     Variables<typename InterpolationTargetTag::vars_to_interpolate_to_target>
         interp_vars(mesh.number_of_grid_points());
 
-    // Clang-tidy wants extra braces for `if constexpr`
-    if constexpr (std::is_same_v<tmpl::list<>, typename InterpolationTargetTag::
-                                                   compute_items_on_source>) {
-      // 1.a Copy the tensors directly into the variables; no need to
-      // make a DataBox because we have no ComputeItems.
+    if constexpr (InterpolationTarget_detail::has_compute_vars_to_interpolate_v<
+                      InterpolationTargetTag>) {
+      // 1a. Call compute_vars_to_interpolate.  Need the source in a
+      // Variables, so copy the variables here.  (This copy would be
+      // unnecessary if we could pass Variables into Events instead of
+      // passing Tensors into Events).
```

Comment:
Why can't you get Variables in Events?

Reply:
Yes you can.  I clarified the comments about this.